cmd/logcli/tail.go

```diff
@@ -59,5 +60,10 @@ func tailQuery() {
 				printLogEntry(entry.Timestamp, labels, entry.Line)
 			}
 		}
+		if len(tailReponse.DroppedEntries) != 0 {
+			for _, d := range tailReponse.DroppedEntries {
+				fmt.Println(d.Timestamp, d.Labels)
```

Comment:
are you just printing then for debug purpose ?

Reply:
Printing it to let the user know we dropped some entries, will add more context to it.

Reply:
yes that was my main concern, it seems it would print a ts and labels out of the blue right now :)